rskj-core/src/main/java/co/rsk/net/eth/RskWireProtocol.java

```diff
@@ -69,7 +69,6 @@ public class RskWireProtocol extends SimpleChannelInboundHandler<EthMessage> imp
     private final SyncStatistics syncStats = new SyncStatistics();
     private final Channel channel;
     private final EthVersion version;
-    private EthState ethState = EthState.INIT;
```

Comment:
Removed, after spotting almost useless usage of this property `ethState` and removing related getters.